elementary/monitor/api/models/models.py

```diff
@@ -54,7 +54,7 @@ def get_models_runs(
                 ModelRunSchema(
                     id=model_run.invocation_id,
                     time_utc=model_run.generated_at,
-                    status=model_run.status,
+                    status=model_run.statuses,
```

Comment:
Any chance this was refactored by mistake?
I think it should still be `model_run.status`?

(and also the ones below)

Reply:
yep, thanks for catching this! fixed